app/ante.go

```diff
@@ -47,21 +55,20 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		ante.NewSetUpContextDecorator(),
 		wasmkeeper.NewLimitSimulationGasDecorator(options.WasmConfig.SimulationGasLimit),
 		wasmkeeper.NewCountTXDecorator(options.TxCounterStoreKey),
-		ante.NewRejectExtensionOptionsDecorator(),
-		ante.NewMempoolFeeDecorator(),
+		ante.NewExtensionOptionsDecorator(options.ExtensionOptionChecker),
```

Comment:
The `MempoolFeeDeductor` has been deleted. Is it now merged inside the `ExtensionOptionsDecorator` together with the `RejectExtensionOptionsDecorator`?

Reply:
It merge into `DeductFeeDecorator` now. See:
https://github.com/cosmos/cosmos-sdk/blob/v0.47.0/UPGRADING.md#go-api-changes

Just realize that we need to have fix to subspace feegrant to have a tx checker as well like:
https://github.com/cosmos/cosmos-sdk/blob/v0.47.0/x/auth/ante/fee.go#L56

I will open a new PR for it.

Reply:
The update is here:
https://github.com/desmos-labs/desmos/pull/1107